app/navigators/TabNavigator.tsx

```diff
@@ -0,0 +1,109 @@
+import { BottomTabScreenProps, createBottomTabNavigator } from "@react-navigation/bottom-tabs"
+import { CompositeScreenProps } from "@react-navigation/native"
+import React from "react"
+import { TextStyle, ViewStyle } from "react-native"
+import { useSafeAreaInsets } from "react-native-safe-area-context"
+import { Icon } from "../components"
+import { translate } from "../i18n"
+import { PlaceholderScreen } from "../screens"
+import { colors, spacing, typography } from "../theme"
+import { AppStackParamList, AppStackScreenProps } from "./AppNavigator"
+
+export type TabParamList = {
+  Schedule: undefined
+  Venue: undefined
+  Explore: undefined
+  Info: undefined
+}
+
+/**
+ * Helper for automatically generating navigation prop types for each route.
+ *
+ * More info: https://reactnavigation.org/docs/typescript/#organizing-types
+ */
+export type TabScreenProps<T extends keyof TabParamList> = CompositeScreenProps<
+  BottomTabScreenProps<TabParamList, T>,
+  AppStackScreenProps<keyof AppStackParamList>
+>
+
+const Tab = createBottomTabNavigator<TabParamList>()
+
+export function TabNavigator() {
+  const { bottom } = useSafeAreaInsets()
+
+  return (
+    <Tab.Navigator
+      screenOptions={{
+        headerShown: false,
+        tabBarHideOnKeyboard: true,
+        tabBarStyle: [$tabBar, { height: bottom + 70 }],
+        tabBarActiveTintColor: colors.text,
+        tabBarInactiveTintColor: colors.text,
+        tabBarLabelStyle: $tabBarLabel,
+        tabBarItemStyle: $tabBarItem,
+      }}
+    >
+      <Tab.Screen
```

Comment:
[nits] thoughts on this being in a map? that way the array can be updated and it updates here?

Reply:
Are you referring to the `<Tab.Screen />` components? Extra name, tx label and icon name to objects in an array and build the children from that?

Reply:
yeah, exactly—it just seems like they're all the same other than the 4 items. something like:
```
const screens = [
  {
    name: "Schedule",
    component: PlaceholderScreen,
    txLabel: "tabNavigator.scheduleTab",
    iconName: "components",
  },
  ...
]
```